src/index.ts

```diff
@@ -22,8 +23,8 @@ const OptionDefaults: IOptions = {
   replaceFences: true,
   rolesAfter: "inline",
   directivesAfter: "block",
-  directives: { ...admonitions, ...images, ...code, ...tables },
-  roles: { ...roles, ...html, ...math }
+  directives: { ...admonitions, ...images, ...code, ...tables, ...mathDriective },
```

Comment:
```suggestion
  directives: { ...admonitions, ...images, ...code, ...tables, ...mathDirective },
```

Reply:
These are getting changed in the next PR as well btw. so didn't worry too much about them here.